src/index.html

```diff
@@ -77,31 +77,70 @@
         </div>
         
         <div class="title">Do you have a living room?</div>
-        <div class="button button--secondary">Add living room</div>
+        <div class="button button--alternative button--disabled button--cool">Add living room</div>
+
+        <div class="title">Do you have a kitchen?</div>
+        <div class="button button--alternative button--alternative button--cool button--large">Add kitchen</div>
         
         <div class="title">Which amenities does your property have?</div>
         <div class="input-container input-container--bordered input-container--focused">
-          <div class="checkbox checkbox--checked"></div>
+          <div class="checkbox">
+             <?xml version="1.0" encoding="utf-8"?>
```

Comment:
You don't need the xml declaration to use svg, remove this and all for this file.